struct Person
{
	1: optional string name;
	2: optional i32 age;
	3: optional bool male;
}



struct ResPerson
{
	1: optional string new_name;
	2: optional bool male;
}
